app/src/common/settings-manager.js

```diff
@@ -37,6 +37,10 @@ const volatiles = {
   }
 };
 
+export const DEFAULT_CROPPER_WINDOW_POSITION = 'center';
+
+const volatiles = JSON.parse(JSON.stringify(DEFAULT_VOLATILES));
```

Comment:
Poor mans deep-clone? 😂

Reply:
A better option is just making `DEFAULT_VOLATILES` a function that returns the object. Then there's no need for cloning as you get a fresh object each time.

Reply:
good point!